cmd/helm/template.go

```diff
@@ -100,6 +100,8 @@ func newTemplateCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 					var manifestsToRender []string
 					for _, f := range showFiles {
 						missing := true
+						// Use linux-style filepath separators to unify user's input path
+						f = filepath.ToSlash(f)
```

Comment:
why is this necessary? This wasn't part of the original code...

https://github.com/helm/helm/blob/df3332e61acf9fea88376102ae3372bf7e6d23b6/cmd/helm/template.go#L195-L231

The assertion on https://github.com/helm/helm/blob/df3332e61acf9fea88376102ae3372bf7e6d23b6/cmd/helm/template.go#L209-L210 means that there should be no need to convert the filepath separator.

Can you please either add a unit test that demonstrates what this is attempting to fix or remove this code. This wasn't part of the original fix.

Reply:
On windows, file path can use ```templates/service.yaml``` or ```templates\service.yaml```

I test two format file path and the result is:
- helm/master version run on windows:
    - use templates/service.yaml ------------------[ fail ]
    - use templates\service.yaml ------------------[ success ]
- My first fix version run on windows:
    - use templates/service.yaml ------------------[ success ]
    - use templates\service.yaml ------------------[ fail ]
- My second fix version run on windows:
    - use templates/service.yaml ------------------[ success ]
    - use templates\service.yaml ------------------[ success ]

And I try to write a unit test to demonstrate this.It works on Windows well,but error on Linux.